vignettes/runif-via-pmap.Rmd

```diff
@@ -0,0 +1,213 @@
+---
+title: "Generate data from different distributions via pmap()"
+author: Safouane Chergui
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(purrr)
+```
+
```

Comment:
I think you need a paragraph that motivate the vignette. Why should someone read it? What will they learn?

Reply:
Added at the top of the vignette